### ISSUE TITLE
Syntax help: how initialize bidimensional array?

### ISSUE THREAD

**Issue by realtebo** [JoshMcguigan/arr_macro#12]
Having 

    pub struct Game {
        pub table: [[CellContent; 3]; 3],
    }

i'd like to initalize using  CellContent::randomized() as source for each 3x3 cell of my table

Is it possible with your crate?

**Comment by tmbull** [comment]
I would like to know the answer to this question as well. I tried `arr![arr![Color::arbitrary(g); 3]; 3]`, but got this error: 

```
error[E0659]: `proc_macro_call` is ambiguous
...
   = note: ambiguous because of a conflict between a macro-expanded name and a less macro-expanded name from outer scope during import or macro resolution
```

The best I could come up with was: 
```
let mut mk_inner = || arr![Color::arbitrary(g); 3];
arr![mk_inner(); 3]
```

**Comment by JoshMcguigan** [comment]
Thanks for the report!

I've published version `0.2.1` which supports nested invocations, via [proc_macro_nested](https://docs.rs/proc-macro-nested/latest/proc_macro_nested/). Let me know if that works for you.

**Comment by tmbull** [comment]
@JoshMcguigan very sorry for the delay, but I've tested the fix and it works for me. Thanks for the quick turnaround!

FWIW, it works with dimensional arrays in general. Ex: 

```
arr![arr![arr![Color::arbitrary(g); 3]; 3]; 6]
```
